### PR TITLE
TOMEE-2291 - Fault Tolerance Microprofile example for @Retry

### PR DESCRIPTION
**Pull request by josehenriqueventura**
I have finished the implementation of mp-faulttolerance-retry

### CODE REVIEW

File: examples/mp-faulttolerance-retry/src/main/java/org/superbiz/rest/WeatherGateway.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    private static final String FORECAST_BUSY_MESSAGE =
+            "Error AccuWeather Forecast Service is busy. Number of Attempts: (%d) \n";
+
+    /**
+     * {@link Retry#maxRetries()}
+     */
+    private static final int DEFAULT_MAX_RETRY = 3;
+
+    private AtomicInteger counterStatusOfDay = new AtomicInteger();
```
[the remark this answers is not in the document]
Comment (josehenriqueventura) [reply examples/mp-faulttolerance-retry/src/main/java/org/superbiz/rest/WeatherGateway.java]: Yes, sure.

File: examples/mp-faulttolerance-retry/src/main/java/org/superbiz/rest/WeatherGatewayBusyServiceException.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.superbiz.rest;
+
+import org.eclipse.microprofile.faulttolerance.exceptions.FaultToleranceException;
+
+public class WeatherGatewayBusyServiceException extends FaultToleranceException {
```
[the remark this answers is not in the document]
Comment (josehenriqueventura) [reply examples/mp-faulttolerance-retry/src/main/java/org/superbiz/rest/WeatherGatewayBusyServiceException.java]: Definitely looks not a good look at this now. Good catch, thanks.

File: examples/mp-faulttolerance-retry/pom.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>examples</artifactId>
```
[the remark this answers is not in the document]
Comment (josehenriqueventura) [reply examples/mp-faulttolerance-retry/pom.xml]: Yes, sure.

**Comment by brunobat** [comment]
Hi @josehenriqueventura you have a conflict. Please rebase your branch with the latest master.

File: examples/mp-faulttolerance-retry/pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (brunobat) [reply examples/mp-faulttolerance-retry/pom.xml]: +1

File: examples/mp-faulttolerance-retry/src/main/java/org/superbiz/rest/WeatherGateway.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (brunobat) [reply examples/mp-faulttolerance-retry/src/main/java/org/superbiz/rest/WeatherGateway.java]: +1

**Comment by josehenriqueventura** [comment]
@brunobat Hi Bruno, could you confirm if it looks good now, please?

**Comment by jeanouii** [comment]
It still has a conflict @josehenriqueventura

**Pull request merged**